Source/WebCore/Modules/modern-media-controls/controls/vision-media-controls.css

```
@@ -40,6 +40,11 @@
     --center-controls-bar-button-zoom: 2;
 }
 
+.media-controls.vision.fullscreen .controls-bar.simple-layout .slider.vision.volume,
+.media-controls.vision.fullscreen .controls-bar.simple-layout .slider.vision.volume * {
```

```suggestion
.media-controls.vision.fullscreen .controls-bar .slider.volume,
.media-controls.vision.fullscreen .controls-bar .slider.volume * {
```

This is probably sufficient.